components/TestimonialList/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Patron from './Patron'
+import { smBreakpoint } from '../../styling/breakpoints'
+import { mediumSpacing, extraSmallSpacing } from '../../styling/sizes'
+import Markdown from '../Markdown'
+
+const PatronContainer = styled.div`
+  margin-bottom: ${mediumSpacing};
+`
+
+const StyledPatron = Patron.extend`
+  @media (max-width: ${smBreakpoint}) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    margin-bottom: ${extraSmallSpacing};
+    text-align: center;
+  }
+`
+
+const PatronList = ({ data: patrons, className }) => (
```

Comment:
Shouldn't that be `TestimonialList`?

Reply:
Yeah, you are right. In the beginning, I named it as a `PatronList` and then renamed the file, but forgot to do with the actual names inside the file.